ajax/libs/approvejs/package.json

```diff
@@ -0,0 +1,32 @@
+{
+  "name": "approvejs",
+  "filename": "approve.min.js",
+  "version": "3.0.1",
+  "description": "A simple JavaScript validation library that doesn't interfere.",
+  "repository": {
+    "type": "git",
+    "url": "git://github.com/CharlGottschalk/approvejs.git"
+  },
+  "keywords": [
+    "validation",
+    "validate",
+    "approve"
+  ],
+  "homepage": "http://charlgottschalk.co.za/projects/approvejs",
+  "author": "Charl Gottschalk (http://charlgottschalk.co.za)",
+  "license": "MIT",
+  "autoupdate": {
+    "source": "git",
+    "target": "git://github.com/CharlGottschalk/approvejs.git",
+    "fileMap": [
+      {
+        "basePath": "dist",
+        "files": [
+          "approve.min.js",
+          "approve.min.js.map",
+          "approve.js"
```

Comment:
@lcd78706 @cdnjs/intern2 @PeterDaveHello 
I'm not sure how to modify it, I think it can be modify to `"**/!(approve.min.gzip*)"`.
Could you provide some ideas?